SU-SWS/stanford_profile
# D8CORE-941: WYSIWYG Typography.

# READY FOR REVIEW

# Summary
- Moves CSS from https://github.com/SU-SWS/stanford_basic/pull/111 as it is coupled to the paragraph type.

# Review By (Date)
- Feb 5th

# Urgency
- Low/Med

# Steps to Test

1. Check out this branch
2. run `npm install` in the stanford_profile_styles directory
3. Run `npm run publish` and ensure no changes to git repo
4. Clear Drupal caches and ensure that styles and stylesheets load on appropriate pages.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- @kerri-augenstein FYI
- @cjwest 
- @jenbreese FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
